appsdk/sdk_test.go

```diff
@@ -271,3 +274,97 @@ func TestApplicationSettingsNil(t *testing.T) {
 		t.Fatal()
 	}
 }
+
+func TestLoadConfigurablePipelineFunctionNotFound(t *testing.T) {
+	sdk := AppFunctionsSDK{
+		LoggingClient: lc,
+		config: common.ConfigurationStruct{
+			Writable: common.WritableInfo{
+				Pipeline: common.PipeLineInfo{
+					ExecutionOrder: "Bogus",
+					Functions:      make(map[string]common.PipelineFunction),
+				},
+			},
+		},
+	}
+
+	appFunctions, err := sdk.LoadConfigurablePipeline()
+	assert.Error(t, err, "expected error for function not found in config")
+	assert.Nil(t, appFunctions, "expected app functions list to be nil")
+}
+
+func TestLoadConfigurablePipelineNotABuiltInSdkFunction(t *testing.T) {
+	functions := make(map[string]common.PipelineFunction)
+	functions["Bogus"] = common.PipelineFunction{}
+
+	sdk := AppFunctionsSDK{
+		LoggingClient: lc,
+		config: common.ConfigurationStruct{
+			Writable: common.WritableInfo{
+				Pipeline: common.PipeLineInfo{
+					ExecutionOrder: "Bogus",
+					Functions:      functions,
+				},
+			},
+		},
+	}
+
+	appFunctions, err := sdk.LoadConfigurablePipeline()
+	assert.Error(t, err, "expected error: Function not a built in sdk function")
```

Comment:
Should also check we got the correct error?

Reply:
Yes